### PR TITLE
Amazing project

### PR DESCRIPTION
**Pull request by sumitisnot4u**

### CODE REVIEW

**Comment by AtieP** [comment]
no

**Comment by resynth1943** [comment]
OMG bro that's so unique and useful and cool thanks

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+// Amazing project
```
Comment (resynth1943) [inline README.md:1]: Really good addition, thanks

**Review by resynth1943 — APPROVED** [review]

**Comment by resynth1943** [comment]
@sumitisnot4u what about the shirt?

**Comment by resynth1943** [comment]
r_1944578_kPa8z

**Pull request closed without merging**